schema/url/pass/plugin-barplot.raml

```diff
@@ -14,8 +14,7 @@ types:
       outputEntityId:
         type: string
       showMissingness:
-        type: string
-        enum: ['TRUE', 'FALSE']
+        type: ShowMissingness
```

Comment:
This only needs to be showMissingnessNoAxes, no? If the plan is to use the histo plot.data function then only pie and histo would use ShowMissingness as opposed to ShowMissingnessNoAxes I'd think

Reply:
yea its a good question. right now the only restriction i put was on where it was or wasnt currently working. but youre right it results in some very arbitrary decisions here... 
i guess if we make a dedicated map histo endpoint then i can apply the two pretty cleanly based on whether its for the map or not. and my current thinking is we should probably have a dedicated map histo anyway... so ill update it